src/funding/card/config.jsx

```diff
@@ -90,85 +82,94 @@ export function getCardConfig() : FundingSourceConfig {
         vendors,
 
         colors: [
-            BUTTON_COLOR.TRANSPARENT
+            BUTTON_COLOR.WHITE,
+            BUTTON_COLOR.BLACK
         ],
 
         secondaryColors: {
-            [ DEFAULT ]: BUTTON_COLOR.TRANSPARENT
+            ...DEFAULT_FUNDING_CONFIG.secondaryColors,
+            [ DEFAULT ]: BUTTON_COLOR.BLACK
         },
 
-        handleClick: true,
-
-        Logo: ({ fundingEligibility, locale, onClick, cardButtonExperiment, nonce }) => {
-            // Testing for Inline Guest black button
-            if (cardButtonExperiment && locale.lang === 'en' && locale.country === 'US' && !__TEST__) {
-                return (
-                    <div>
-                        <style
-                            nonce={ nonce }
-                            innerHTML={ `
-                                #black-credit-card-button {
-                                    background: #000;
-                                    color: #fff;
-                                    font-size: 1.15em;
-                                    border-radius: 4px !important;
-                                }
-                            ` } />
-                        <button
-                            type="button"
-                            id="black-credit-card-button"
-                            onClick={ event => onClick(event) }
-                            class={ `${ CLASS.BUTTON }` }
-                            { ...{
-                                [ ATTRIBUTE.FUNDING_SOURCE ]: FUNDING.CARD
-                            } }>
-                            Debit or Credit Card
-                        </button>
-                    </div>
-                );
-            }
-
-            let maxCards = 4;
-
-            // $FlowFixMe
-            if (maxCardForCountry[locale.country]) {
-                maxCards = maxCardForCountry[locale.country];
-            }
-
-            return CARD_PRIORITY.map(name => {
-
-                const cardEligibility = fundingEligibility[FUNDING.CARD];
-
-                if (!cardEligibility || !cardEligibility.vendors || !cardEligibility.vendors[name] || !cardEligibility.vendors[name].eligible) {
-                    return null;
-                }
+        logoColors: {
+            [ BUTTON_COLOR.WHITE  ]:  BUTTON_COLOR.BLACK,
+            [ DEFAULT ]:              BUTTON_COLOR.WHITE
+        },
 
-                const vendorConfig = vendors[name];
+        handleClick: true,
 
-                if (!vendorConfig) {
-                    return null;
-                }
+        Logo: ({ onClick, nonce, content, logoColor, locale = {} }) => {
+            const { lang } = locale;
+            const isRTLLanguage = lang === 'he' || lang === 'ar' || lang === 'fa';
```

Comment:
Can we:

1. Separate `isRTLLanguage` into a utility somewhere?
2. Use constants here e.g. `LANG.HE`